examples/00_tutorials/plot_single_subject_single_run.py

```diff
@@ -148,42 +168,80 @@
 # then a :term:`contrast` that makes the difference between these conditions.
 
 from numpy import array
+
 conditions = {
-    'active': array([1., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.,
-                     0.]),
-    'rest':   array([0., 1., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.,
-                     0.]),
+    "active": array(
```

Comment:
It might just be me, but I don't think this will render in a super pretty way in the example...
Maybe something like this would be more compact and clear:

```python
conditions = {"active": np.zeros(16), "rest": np.zeros(16)}
conditions["active"][0] = 1
conditions["rest"][1] = 1
```

WDYT?

Reply:
good idea

should be fixed now